templates/superset.service.j2

```diff
@@ -12,7 +12,7 @@ Environment="PATH={{ superset_virtualenv_path }}/bin:$PATH"
 Environment="PYTHONPATH={{ superset_python_path }}"
 User={{ superset_user }}
 Type=simple
-ExecStart={{ superset_virtualenv_path }}/bin/superset runserver -p {{ superset_port }} -w {{ superset_workers }}
+ExecStart={{ superset_virtualenv_path }}/bin/superset run -p {{ superset_port }} --with-threads
```

Comment:
@nateMJ with the new command, are we able to specify the number of threads/workers?

Reply:
Unfortunately not although it's possible using gunicorn. I'm not sure if we want to use that over flask commands

Reply:
Makes sense. Just checked and the superset_workers variable is used in the systemd unit file for the service.